catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/impl/TestOpenSearchSource.java

```diff
@@ -406,6 +410,7 @@ public void testQueryById() throws UnsupportedQueryException, IOException {
 
     // then
     assertThat(response.getHits(), is(1L));
+    assertSourceId(response);
```

Comment:
❓Instead of adding this check in all/most unit tests, should we have a single test that makes sure the source ID is set, and one that ensures it gets added when it's missing? Adding the source ID check makes it look like that's the only thing we need to verify, when in reality we should make sure _all_ the metacard attributes have been properly set, not just the source ID.

Reply:
I'll make a single test for asserting the source id.